.github/workflows/ci.yml

```diff
@@ -85,7 +85,7 @@ jobs:
           pip install .
       - name: Install test dependencies
         run: |
-          pip install pytest pytest-lazy-fixture pipdeptree
+          pip install pytest pytest-lazy-fixture pipdeptree hypothesis
```

Comment:
hypothesis is being added as a specific test dependency in the test-without-extras-min-versions